swagger_gen/lib/schema.py

```diff
@@ -190,7 +190,7 @@ def _create_endpoint_definition(
             # Add generic metadata if it's not defined on the route
             else:
                 defaults = self._get_default_metadata()
-                method_definition = method_definition | defaults
+                method_definition = method_definition or defaults
```

Comment:
This is actually merging the dictionaries, I may need to put a constraint on the package to enforce Python 3.9+ since there are some newer features in the code